### DISCUSSION TITLE
V3: morph-markers also injected in the middle of HTML-Tags

### DISCUSSION THREAD

**Discussion by renky** [livewire/livewire#5844] [7. Bugs (archived)]
Livewire adds its morph marking in between html-tags

```
<!-- __BLOCK__ -->            <th scope="col">
<div
class="py-3 px-6 flex justify-between items-center w-full"
<!-- __ENDBLOCK__ -->
>
<div>ID</div>
 
<!-- __BLOCK__ --> <!-- __ENDBLOCK__ -->
</div>
</th>
```

this happens, because @if / @endif is used in html-blocks, but thats indeed not disallowed. I don't know how much this inject-morph-markers setting optimizes the morphing - but if this is really meassurable, it would be great to be also able using it with blade-directives in the middle of tags. Maybe it would be possible to detect if it is inside a tag, and then not set this block.

**Comment by joshhanley** [comment]
@renky thanks for reporting! Any chance you could open a PR with a failing test? Thanks!

**Comment by zepfietje** [comment]
Probably related: https://github.com/filamentphp/filament/pull/7099.

**Comment by renky** [comment]
seems to be fixed
